packages/mod.go

```diff
@@ -37,6 +37,18 @@ func (m Mod) ExtractPurlsFromManifest() (purls []string) {
 	return purls
 }
 
+func (m Mod) ExtractPurlsFromManifestForIQ() (purls []string) {
+	for _, s := range m.ProjectList.Projects {
+		if len(s.Version) > 0 { // There must be a version we can use
+			var version = s.Version
+			version = strings.Replace(version, "+incompatible", "", -1)
```

Comment:
So this and OSS index version of this method the only real different in what is being replaced. 

IQ = we replace "+incompatible" with ""
OSS = we repalce "v" with ""

Is it really true that each one can handle different version strings. Or is each on currently handling a different edge case that we might want for both??

Reply:
There is a healthy internal convo going about this here predicament, right now :)

Right now OSS Index prefers no `v` but I found a case today where it took one, which caused the conversation.

IQ demands a `v` at the moment.

I stripped `+incompatible` because it really isn't SemVer, afaik. I did however notice it didn't cause an issue with OSS Index, and I think it has to do with how OSS Index does version ranges.

Reply:
o neat well....
![godspeed](https://user-images.githubusercontent.com/1099906/71705381-7d1f0b00-2dad-11ea-8e0b-f44b5b3a8afb.gif)

Reply:
Outcome of the convo: strip the `v` for OSS Index, keep it for `IQ`. Pretty sure I reduce some of the code duplication by just doing a method that calls the other method with the data sanitized or something akin, OR we could create a new method that takes a function as a parameter, and then runs that function on the version. WDYT?

Reply:
Keeping this as it is for now.